portfolio/src/main/webapp/index.html

```diff
@@ -78,12 +78,33 @@ <h3>Comments</h3>
       <div class="comments" id="comments">
         <p id="no-comments">No comments yet</p>
         <form action="/comments" method="POST" id="main-form">
-          <input type="text" name="username" placeholder="Your username">
-          <input type="text" name="text" placeholder="Write your comment here">
+          <input type="text" name="username" required placeholder="Your username">
+          <input type="text" name="text" required placeholder="Write your comment here">
           <input type="submit">
         </form>
         <script>
-          fetchComments();
+          fetchComments().then(() => {
```

Comment:
This sounds like it will work for sure. But I feel like we are going round the world. Can we instead just set this validator for every input we generate?